assembl/static2/js/app/actions/adminActions/index.js

```diff
@@ -31,6 +31,16 @@ export const createNewThematic = id => ({
   type: 'CREATE_NEW_THEMATIC'
 });
 
+export const moveThematicUp = id => ({
```

Comment:
not really related to this PR but it would be nice to move survey stuff in a new `adminAction/survey.js` file and to add flow